docs/pages/includes/application-access/aws-database-create-iam-role.mdx

```diff
@@ -0,0 +1,79 @@
+The first step is to create an IAM role for your Teleport users to assume.
```

Comment:
I think it would be good to add another sentence or two of context about when users assume this role and what users assume this role in order to do. 

If that context differs too much between pages that include this partial, we could remove this paragraph from the partial and add it to the including pages instead.